lua/auto-session.lua

```diff
@@ -480,6 +480,19 @@ function AutoSession.RestoreSessionFromFile(session_file)
   AutoSession.RestoreSession(string.format(AutoSession.get_root_dir() .. "%s.vim", session_file:gsub("/", "%%")))
 end
 
+--
+-- Refresh syntax highlighting and file trees
+local function post_restore_refresh()
```

Comment:
This is what fixes the syntax highlighting. I moved it to after the restore session, so that it's always triggered. Although, I'm not sure if the problem exists other than in during the `autocmd` callbacks? So, it might be better to move it back there if not.

Reply:
Interesting, I hadn't noticed problems with this. But this should be fine anyway 👍